tests/unit_tests/test_wave_06.py

```diff
@@ -75,17 +75,31 @@ def test_swap_best_by_category():
         my_priority="Clothing",
         their_priority="Decor"
     )
-
-    raise Exception("Complete this test according to comments below.")
-    # *********************************************************************
-    # ****** Complete Assert Portion of this test **********
-    # *********************************************************************
+    # Assert 
     # Assertions should check:
     # - That the results is truthy
+    assert result == True
     # - That tai and jesse's inventories are the correct length
+    assert len(tai.inventory) == 3
+    assert len(jesse.inventory) == 3
     # - That all the correct items are in tai and jesse's inventories, including the items which were swapped from one vendor to the other
+    assert item_a and item_b and item_f in tai.inventory
+    assert item_c and item_d and item_e in jesse.inventory 
+    '''
+    Hi Audrey, I am wondering which one is the better way of writing the assertions! Thank you :) 
+
+    assert item_a and item_b and item_f in tai.inventory
+    assert item_c and item_d and item_e in jesse.inventory 
+        or 
+    assert item_a in tai.inventory
+    assert item_b in tai.inventory
+    assert item_c not in tai.inventory
+    assert item_d in jesse.inventory
+    assert item_e in jesse.inventory
+    assert item_f not in jesse.inventory
+    '''
```

Comment:
Great question Hailey! I prefer writing the asserts individually for each item so in the event that my code causes an assertion error, we can see the exact item that failed the test.

Let's talk more about this during our 1:1!